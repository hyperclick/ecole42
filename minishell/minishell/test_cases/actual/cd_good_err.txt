-minishell: cd: OLDPWD not set
