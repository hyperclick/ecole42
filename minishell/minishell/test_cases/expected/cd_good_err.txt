sh: line 0: cd: OLDPWD not set
